btree/btree.c

```diff
@@ -1202,6 +1216,18 @@ static int node_valsize(const struct nd *node)
 	M0_PRE(node_invariant(node));
 	return (node->n_type->nt_valsize(node));
 }
+
+static bool  node_underflow(const struct nd *node, bool predict)
```

Comment:
Please write some comment to hint what predict means. This will help the caller determine what parameter (true/false) to send here.

Reply:
okay